### PR TITLE
Fix vision variance copying in mavlink message

### PR DESCRIPTION
**Pull request by kamilritz**
The copying of the vision covariances into the mavlink message was not done properly. The index variable `count` was incremented twice.

### CODE REVIEW

File: src/gazebo_mavlink_interface.cpp
Comment (TSC21) [inline src/gazebo_mavlink_interface.cpp]:
```suggestion
	  count++;
```

**Review by TSC21 — APPROVED** [review]
Nice catch! Just a minor indentation a should be good to go.

File: src/gazebo_mavlink_interface.cpp
Comment (kamilritz) [reply src/gazebo_mavlink_interface.cpp]: Good catch as well

**Comment by kamilritz** [comment]
@TSC21 The macOS tests seem to be failing. I guess this is not related to this PR.

**Comment by kamilritz** [comment]
@TSC21 Can we get this is? I will need correct velocity variances for the CI test.

**Comment by TSC21** [comment]
I can't seem to get it merged. States that the required statuses must pass before merging. Must be another Github trick. @Jaeyoung-Lim are you able to merge this?

**Comment by kamilritz** [comment]
As a maintainer one could make the tests temporarily not a requirement.
The better way would be to fix the macOS build.

**Comment by Jaeyoung-Lim** [comment]
@TSC21 No, I don't have the rights

**Comment by TSC21** [comment]
> As a maintainer one could make the tests temporarily not a requirement.
> The better way would be to fix the macOS build.

No that's not the problem. I can usually merge even if the test doesn't pass. This is a bug in Github (already got these before).

**Comment by kamilritz** [comment]
Are you sure? There is a [setting](https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks) for this.

**Comment by TSC21** [comment]
> Are you sure? There is a [setting](https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks) for this.

Yes I am sure. Even with the requirements, if you are an admin you can merge it. This is a bug. This is currently what shows up to me:

Screenshot from 2020-05-02 10-56-17

It's like it is stuck on checking the requirements. Can you try maybe doing a force push to see if this moves?

**Update:** It's not a bug, it's just me that do not have admin privileges.

**Comment by kamilritz** [comment]
There is even a setting (point 9 in the link) that can be set so that those tests need to pass also for administrators for merging. I don't have access to the settings so I can't check.

**Comment by TSC21** [comment]
> There is even a setting (point 9 in the link) that can be set so that those tests need to pass also for administrators for merging. I don't have access to the settings so I can't check.

OK I remove what I said: I just approved https://github.com/PX4/sitl_gazebo/pull/476, which was missing an approval to be merged, and the same appeared. Someone changed the permissions to `sitl_gazebo` as I don't seem to have admin privileges anymore. So there's nothing I can do.

**Comment by TSC21** [comment]
@kamilritz the problem in the MacOS build is not related with the containers, as those are not used on the MacOS pipeline. @Jaeyoung-Lim already tried to work on a fix in #473 but the problem seems to come from the OSRF formulas. So there is no fix we can do on this repo.

**Comment by kamilritz** [comment]
Ok, thanks. Could we disable the macOS test until there is a fix for this? This is really blocking.

**Comment by TSC21** [comment]
> Ok, thanks. Could we disable the macOS test until there is a fix for this? This is really blocking.

I am against disabling it but I am not against merging this PR (without the last commit). Though, someone with admin previleges needs to merge it.

**Comment by kamilritz** [comment]
Ok, removed the last commit. Do you know who has admin rights?

**Review by Jaeyoung-Lim — APPROVED** [review]
Thanks!

**Comment by TSC21** [comment]
Things rolling now. Thanks @kamilritz!

**Pull request merged**
